internal/cli/cmd/cluster/registry.go

```diff
@@ -593,35 +593,16 @@ Alternatively, use --repository and --digest flags.`,
 
 	repositoryFlag := cmd.Flags().String("repository", "", "Repository name")
 	digestFlag := cmd.Flags().String("digest", "", "Image digest")
-	expiresAt := cmd.Flags().String("expires-at", "", "Set new absolute expiry time in RFC3339 format (e.g., 2024-12-31T23:59:59Z)")
-	extendBy := cmd.Flags().Duration("extend-by", 0, "Extend current expiry by this duration (e.g., 720h for 30 days)")
-	ensureMinimum := cmd.Flags().Duration("ensure-minimum", 0, "Ensure at least this much time remains before expiry (e.g., 168h for 7 days)")
-	output := cmd.Flags().StringP("output", "o", "table", "Output format: table, json")
-
-	cmd.RunE = fncobra.RunE(func(ctx context.Context, args []string) error {
-		repository := args[0]
-		digest := args[1]
 
-		// Validate mutually exclusive flags
-		flagCount := 0
-		if *expiresAt != "" {
-			flagCount++
-		}
-		if *extendBy > 0 {
-			flagCount++
-		}
-		if *ensureMinimum > 0 {
-			flagCount++
-		}
+	expireAt := cmd.Flags().String("expire-at", "", "Set new absolute expiry time in RFC3339 format (e.g., 2024-12-31T23:59:59Z)")
+	ensureMinimum := cmd.Flags().Duration("ensure-minimum", 0, "Ensure at least this much time remains before expiry (e.g., 168h for 7 days)")
 
-		if flagCount == 0 {
-			return fnerrors.BadInputError("must specify one of --expires-at, --extend-by, or --ensure-minimum")
-		}
+	cmd.MarkFlagsOneRequired("expire-at", "ensure-minimum")
+	cmd.MarkFlagsMutuallyExclusive("expire-at", "ensure-minimum")
```

Comment:
In theory these two are not mutually exclusive. Shall we allow a user to specify both?